sqldelight-gradle-plugin/src/test/integration-postgresql/src/test/kotlin/com/squareup/sqldelight/postgresql/integration/PostgreSqlTest.kt

```diff
@@ -0,0 +1,48 @@
+package com.squareup.sqldelight.postgresql.integration
+
+import com.google.common.truth.Truth.assertThat
+import com.squareup.sqldelight.sqlite.driver.JdbcDriver
+import java.sql.Connection
+import java.sql.DriverManager
+import org.junit.After
+import org.junit.Before
+import org.junit.Test
+
+class PostgreSqlTest {
+
+  val connPostgres = DriverManager.getConnection("jdbc:tc:postgresql:9.6.8:///postgres")
+  lateinit var connMyDb: Connection
+  lateinit var driver: JdbcDriver
+  lateinit var database: MyDatabase
+
+  @Before
+  fun before() {
+    connPostgres.prepareStatement("CREATE DATABASE my_db;").execute()
```

Comment:
FYI you don't need to create a DB yourself.
In fact, in this test, you're creating *two* PostgreSQL containers, where one is no-op :D 
It happens because Testcontainers uses the provided URL's hash to assign instances, so you have two:

1. `jdbc:tc:postgresql:9.6.8:///postgres`
2. `jdbc:tc:postgresql:9.6.8:///my_db`

consider removing `connPostgres` and associated code

Reply:
Thanks again! This was a remnant of trying to manually deal with the PostgreSQL lifecycle. Fixed in https://github.com/cashapp/sqldelight/pull/1681/commits/791b48e707cc8c12431c3637b132b4f0e11131cc.